packages/flutter_tools/test/general.shard/application_package_test.dart

```diff
@@ -213,6 +213,54 @@ void main() {
       expect(data.launchableActivityName, 'io.flutter.examples.hello_world.MainActivity2');
     });
 
+
+    testWithoutContext('Parses manifest with an activity-alias that has enabled set to true, action set to android.intent.action.MAIN and category set to android.intent.category.LAUNCHER and Activity with no action nor category', () {
```

Comment:
[This is just a general question]

I feel like this test name is really long. Not sure if it's OK in this project.
I'd consider renaming these test cases to sth like "Parses manifest with an activity-alias (1)" and (2) and so on.
I'm curious what others think about it.

Reply:
I followed the other test names convention in this file.